tests/cluster/delta/test_sparkexecutor.py

```diff
@@ -7,30 +7,36 @@
 from tests.cluster.delta.SparkExecutor import SparkSqlExecutor
 
 
+def delete_all_databases():
```

Comment:
If we parralize test jobs - this will end up intefere with other tests. 
Also - in a true dataplatform setup - this CAN NOT be used since it will drop true databases